### DISCUSSION TITLE
Show warning when instantiating variable with same name as another

### DISCUSSION THREAD

**Discussion by fransab** [gama-platform/gama#69] [Ideas]
Hey everyone,
   I've been using GAMA for the past few weeks for a project, and I struggle a bit to adapt to GAMA nomenclature. There are many variables defined implicitly (like time,cycle...) and it's hard to understand some times which names are forbidden. 
As an example, I also had trouble debugging an issue, because I named an agent variable with the same name as another species defined in another file that was imported, and didn't notice the name was already used. 
It could be useful to find out which variables, species, ... are accessible in a file, and to display a warning whenever we try to instantiate an agent, variable... with the same name.

**Comment by lesquoyb** [comment]
this has been fixed since then
